src/style.css

```diff
@@ -0,0 +1,126 @@
+:root {
+--main-bg-color: #fff;
+--dark-txt-color: #060b35;
+--grey-txt-color: #616070;
+--button-color: #00acdc;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  background: var(--main-bg-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0;
+  padding: 32px 16px 16px;
+}
+
+.card__imac-image {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  color: var(--dark-txt-color);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 0 4px;
+}
+
+.card__product-code {
+  color: var(--grey-txt-color);
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin: 24px 0 16px;
+  height: 18px;
+}
+
+.price-text {
+  color: var(--grey-txt-color);
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.price-amount {
+  color: var(--dark-txt-color);
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__buy-button {
+  width: 100%;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  color: var(--main-bg-color);
+  text-align: center;
+  text-decoration: none;
+  background: var(--button-color);
+  border-radius: 5px;
+}
+
+.card__buy-button:hover {
+  background: var(--main-bg-color);
+  border: 1px solid var(--button-color);
+  color: var(--button-color);
+}
+/* #endregionCard */
+
+/* #regionStars */
+.stars {
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  margin: 16px 0 0;
+}
+
+.stars__stars-4 {
+  display: flex;
+  align-items: center;
+}
+
+.star {
+  width: 16px;
+  height: 16px;
+  background: url(images/star.svg) center no-repeat;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
you should have styles for all modificators, from 1 to 5.

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/d1f60966-f43a-4445-ae86-e66d41f6d8b6)

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/55200530/b6964535-353a-43c7-b764-4c0c59c4ba47)

Reply:
also would be good to use scss with nesting